frontend/notarial-blockchain-ui/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
is necessary?

Reply:
yep, without this prettier might not run properly in JetBrains' IDE's